service/worker/migration/force_replication_workflow_test.go

```diff
@@ -95,7 +95,7 @@ func TestForceReplicationWorkflow(t *testing.T) {
 	assert.True(t, status.TaskQueueUserDataReplicationStatus.Done)
 	assert.Equal(t, "", status.TaskQueueUserDataReplicationStatus.FailureMessage)
 	assert.Equal(t, int64(10), status.TotalWorkflowCount)
-	assert.Equal(t, int64(0), status.ReplicatedWorkflowCount)
+	assert.Equal(t, int64(4), status.ReplicatedWorkflowCount)
```

Comment:
Why is this 4? Shouldn't ReplicatedWorkflowCount be equal to TotalWorkflowCount?